server/controllers/async-controllers.js

```diff
@@ -1,35 +1,40 @@
 import {getSeries} from '../services/wordpress';
 import {PromoListFactory} from '../model/promo-list';
-import {getForwardFill, getUnpublishedSeries} from '../model/series';
+import {getForwardFill} from '../model/series';
 import {getSeriesColor} from '../data/series';
 import {createNumberedList} from '../model/numbered-list';
 import {getLatestInstagramPosts} from '../services/instagram';
+import {getArticleSeries} from '../services/prismic';
 
-export const seriesNav = async(ctx, next) => {
+const getSeriesData = async(ctx) => {
   const {id} = ctx.params;
```

Comment:
Could we leave a note here about the inefficiency of this?
We make a WP call, if we fail, then we make two other API calls to Prismic.

Just so that we're aware of what we doing.